docs/img/LATENCY_VS_THROUGHPUT.svg

```diff
@@ -1,60 +1,378 @@
-</defs>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Cannot see any reference to this file in other changed parts? Is it really a part of the proposed changes?

Reply:
This is a fix for the picture for the Performance Optimizations Guide, not related to the proposed changes directly